fault.c

```diff
@@ -157,24 +233,289 @@ static int kibosh_fault_read_delay_check(struct kibosh_fault_read_delay *fault,
     if (strncmp(path, fault->prefix, strlen(fault->prefix)) != 0) {
         return 0;
     }
-    milli_sleep(fault->delay_ms);
+    if (strlen(fault->file_type) > strlen(path) ||
```

Comment:
file_type is intended to specify the extension of the file. In Kibosh, I match the file path to the file_type from right to left, so this can be interpreted as a file path suffix as well.